README.md

```diff
@@ -103,4 +103,11 @@ You can run tests locally using:
 python -m pytest
 ```
 
-
+You can configure a different base url with:
```

Comment:
Maybe mark this as "for internal use" given this is mainly a public facing package?

Reply:
OpenAI clients have an option to specify api base url, I think it's just good SDK etiquette. Also, with private cloud deployment, its reasonable customers will want to point to their own servers.